infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -203,109 +172,117 @@ private CorfuRuntime getNewCorfuRuntime() {
         return runtime;
     }
 
-    private boolean isBootstrapped(CorfuMsg msg) {
+    private boolean isBootstrapped(RequestMsg req) {
         if (serverContext.getManagementLayout() == null) {
-            log.warn("Received message but not bootstrapped! Message={}", msg);
+            log.warn("Received message but not bootstrapped! Message={}", req.getHeader());
             return false;
         }
+
         return true;
     }
 
     /**
      * Forward an orchestrator request to the orchestrator service.
      *
-     * @param msg corfu message containing ORCHESTRATOR_REQUEST
-     * @param ctx netty ChannelHandlerContext
-     * @param r   server router
+     * @param req  a message containing an ORCHESTRATOR request
+     * @param ctx  the netty ChannelHandlerContext
+     * @param r    the server router
      */
-    @ServerHandler(type = CorfuMsgType.ORCHESTRATOR_REQUEST)
-    public synchronized void handleOrchestratorMsg(@Nonnull CorfuPayloadMsg<OrchestratorMsg> msg,
-                                                   @Nonnull ChannelHandlerContext ctx,
-                                                   @Nonnull IServerRouter r) {
-        log.debug("Received an orchestrator message {}", msg);
-        orchestrator.handle(msg, ctx, r);
+    @RequestHandler(type = PayloadCase.ORCHESTRATOR_REQUEST)
+    public synchronized void handleOrchestratorMsg(RequestMsg req, ChannelHandlerContext ctx, IServerRouter r) {
+        log.debug("handleOrchestratorMsg: message:{}", req.getPayload().getOrchestratorRequest());
+        orchestrator.handle(req, ctx, r);
```

Comment:
I just noticed some old RPC handlers have Nonnull constraint of their arguments (but some don't, also for other servers...), will it be better if we add Nonnull consistently to all the arguments? 
Currently I cannot image any case where the argument is null but this might be helpful for debugging sometime in the future?

Reply:
Those annotation can improve code inspection for inferring nullity﻿

Reply:
Done. Added the @Nonnull annotations to the handler methods.